### ISSUE TITLE
Project templates

### ISSUE THREAD

**Issue by BabisK** [nephila/python-taiga#38]
Hello,

I've been using this fine library for an internal project and I need to define project templates. I've done some work and I've added some basic models that create an empty template. I will add some more functionality to create roles, points etc in the template definition.

I'd like to merge this here if you are interested in templates. What do you think?

**Comment by astagi** [comment]
Hello!

Yes please feel free to open a PR and I'll review the code :smile: 

Thank you for your contribution!

**Comment by astagi** [comment]
@BabisK any update on this?

**Comment by BabisK** [comment]
Yes, I have concluded work, added all parameters of project templates and tested it on a migration to taiga which is now being used without any problems. The thing is I haven't had the time to unit test it. And as it is a rather large addition (project template contains almost all project fields) this will take some time.

**Comment by astagi** [comment]
sounds great! thank you for the update!!!

**Comment by astagi** [comment]
any update there @BabisK ? :)

**Comment by yakky** [comment]
@BabisK could you share what you have done so far anyway? I think we can help with tests to get them merged
